docker/docker_files/Dockerfile-indexer

```diff
@@ -1,5 +1,6 @@
 FROM promnesia:latest
 
 RUN apt-get update && apt-get install -y cron
+RUN pip install fastapi uvicorn
```

Comment:
hmm, I don't think it's necessary since this Dockerfile is only indexing (e.g. it didn't have `hug` dependency previously either)

Reply:
makes sense -- sorry :)